### PR TITLE
Grammatical Fix

### LINKED ISSUE #12300: Grammatical
This is not really a bug, but I don't know where else to point it out.

The motorBike checklist should read "an engines attribute"  not "a engines attribute"

Challenge [Declare JavaScript Objects as Variables](https://www.freecodecamp.com/challenges/declare-javascript-objects-as-variables#?solution=%0Avar%20car%20%3D%20%7B%0A%20%20%22wheels%22%3A4%2C%0A%20%20%22engines%22%3A1%2C%0A%20%20%22seats%22%3A5%0A%7D%3B%0A%0Avar%20motorBike%20%3D%20%7B%0A%0A%0A%20%20%2F%2F%20Only%20change%20code%20below%20this%20line.%0A%20%20%22wheels%22%3A2%2C%0A%20%20%22engines%22%3A1%2C%0A%20%20%22seats%22%3A1%0A%7D%3B%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript

var car = {
  "wheels":4,
  "engines":1,
  "seats":5
};

var motorBike = {


  // Only change code below this line.
  "wheels":2,
  "engines":1,
  "seats":1
};

```

### PR DESCRIPTION
**Pull request by ryzokuken**
Fixes #12300 

#### Pre-Submission Checklist

- [ ] Your pull request targets the `staging` branch of FreeCodeCamp.
- [ ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [ ] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [ ] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [ ] Tested changes locally.
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX

### CODE REVIEW

**Comment by ryzokuken** [comment]
There was an issue with `MakeFriendlyDate()`, I suppose, but the simple commit does not even modify such a method.

**Comment by Bouncey** [comment]
@ryzokuken please see #12305 

It is not an issue with your PR

Thanks for contributing :+1:

File: seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
       "title": "Declare JavaScript Objects as Variables",
       "description": [
         "Before we dive into Object Oriented Programming, let's revisit JavaScript objects.",
-        "Give your <code>motorBike</code> object a <code>wheels</code>, <code>engines</code> and <code>seats</code> attribute and set them to numbers."
+        "Give your <code>motorBike</code> object <code>wheels</code>, <code>engines</code>, and <code>seats</code> attributes and set each to a number"
```
Comment (Bouncey) [inline seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json]: Comma after `<code>engines</code>` is not required

**Review by Bouncey — CHANGES_REQUESTED** [review]

**Review by systimotic — CHANGES_REQUESTED** [review]
This looks great! If you could add back the period at the end of the sentence and remove the ever-so-controversial Oxford comma as requested by Bouncey, this should be good to merge! Nice work 👍

**Comment by raisedadead** [comment]
@ryzokuken thanks a lot for your change, but looks like you have made a change on GitHub interface, can you close this and make a fresh request from a local copy? This will let you address the review comments.

Have a look at our [Guidelines for Contributing](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md).

Reach us out on the [Contributors Chat room](https://gitter.im/FreeCodeCamp/Contributors), if you need any assistance.

**Comment by ryzokuken** [comment]
@raisedadead Yes, I would love to. I am setting up the project locally. This might take a little time though, and I'm sorry for that.

**Comment by raisedadead** [comment]
@ryzokuken sure no problem.

**Comment by Bouncey** [comment]
Time is not an issue here @ryzokuken :+1: 

Thank you for taking the time to fix this

**Pull request closed without merging**